### PR TITLE
Browser forwarding

### PR DESCRIPTION
**Pull request by esinarta**
implemented useBrowserSender hook to support forwarding on the browser, dynamically imported useNodeSender/useBrowserSender in useForwarder, tested functionality and webhooks are successfully forwarded using the browser

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -21,7 +17,5 @@
     "baseUrl": "src",
     "noFallthroughCasesInSwitch": true
   },
-  "include": [
-    "src"
-  ]
+  "include": ["src"]
```
Comment (johnnyoshika) [inline tsconfig.json]: It looks like we have different prettier settings for json files. Can you revert this change on this file? Otherwise, we'll keep overwriting each other's changes.

**Review by johnnyoshika — CHANGES_REQUESTED** [review]
Great work! Just a minor formatting reversal on the tsconfig, but otherwise, it's good to go.

File: tsconfig.json (hunk 1 above)
Comment (esinarta) [reply tsconfig.json]: tsconfig formatting has been corrected

**Comment by johnnyoshika** [comment]
@esinarta Thank you! 👏
